src/java/org/apache/cassandra/metrics/RatioGaugeSet.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.metrics;
+
+import java.util.function.DoubleSupplier;
+
+import com.codahale.metrics.Metered;
+import com.codahale.metrics.RatioGauge;
+import org.apache.cassandra.metrics.CassandraMetricsRegistry.MetricName;
+
+import static org.apache.cassandra.metrics.CassandraMetricsRegistry.Metrics;
+
+public class RatioGaugeSet
+{
+    public final RatioGauge oneMinuteRatio;
+    public final RatioGauge fiveMinuteRatio;
+    public final RatioGauge fifteenMinuteRatio;
+    public final RatioGauge allTimeRatio;
+
+    public RatioGaugeSet(Metered numerator, Metered denominator, MetricNameFactory factory, String namePattern)
+    {
+        this.oneMinuteRatio = ratioGauge(factory.createMetricName(String.format(namePattern, "OneMinuteRatio")), numerator::getOneMinuteRate, denominator::getOneMinuteRate);
+        this.fiveMinuteRatio = ratioGauge(factory.createMetricName(String.format(namePattern, "FiveMinuteRatio")), numerator::getFiveMinuteRate, denominator::getFiveMinuteRate);
+        this.fifteenMinuteRatio = ratioGauge(factory.createMetricName(String.format(namePattern, "FifteenMinuteRatio")), numerator::getFifteenMinuteRate, denominator::getFifteenMinuteRate);
+        this.allTimeRatio = ratioGauge(factory.createMetricName(String.format(namePattern, "AllTimeRatio")), numerator::getMeanRate, denominator::getMeanRate);
```

Comment:
```suggestion
        this.allTimeRatio = ratioGauge(factory.createMetricName(String.format(namePattern, "MeanRatio")), numerator::getMeanRate, denominator::getMeanRate);
```
...or do you think "mean" is to vague in this context?